src/test/java/software/amazon/smithy/lsp/SmithyInterfaceTest.java

```diff
@@ -105,10 +105,16 @@ public void runValidators() throws Exception {
         assertTrue(result.isRight());
         List<ValidationEvent> validationEvents = result.getRight().getValidationEvents();
         assertFalse(validationEvents.isEmpty());
-        assertEquals(
-            "Proto index 1 is used muliple times in members name,age of shape (structure: `some.test#MyStruct`).",
-                validationEvents.get(0).getMessage()
-        );
+
+        String expectedMessage = "Proto index 1 is used muliple times in members name," +
+                "age of shape (structure: `some.test#MyStruct`).";
+        Optional<ValidationEvent> matchingEvent = validationEvents.stream()
+                .filter(ev ->ev.getMessage().equals(expectedMessage)).findFirst();
```

Comment:
nit: I think you could a hamcrest matcher (or `orElseThrow`) to make this a little cleaner
```
assertThat(validationEvents, hasItem(hasProperty("message", equals(expectedMessage)))
```
or
```
.orElseThrow(() -> new AssertionError(...))
```

Reply:
I would if we used hamcrest in the language server. Will be adding it in the future though.